docs/architecture/004-etl-pipeline.rst

```diff
@@ -1,412 +0,0 @@
-0004 - ETL Pipelne
```

Comment:
Actually that is a decription of this module: https://github.com/locationtech/geotrellis/tree/master/spark-pipeline/src

Which inspired by https://pdal.io/quickstart.html

Which leads us closer to resolving https://github.com/locationtech/geotrellis/issues/1302 if we want it. 

Should not we remove module first than?

Reply:
This is a duplicate file of https://github.com/locationtech/geotrellis/blob/master/docs/architecture/005-etl-pipeline.rst

It remains in the docs: https://geotrellis.readthedocs.io/en/latest/architecture/005-etl-pipeline.html

Reply:
Ouch haven't noticed that, thanks!